src/LibraryManager.Vsix/UI/Models/InstallDialogViewModel.cs

```diff
@@ -376,9 +382,9 @@ private bool CanInstallPackage()
                 await manifest.SaveAsync(_configFileName, CancellationToken.None).ConfigureAwait(false);
 
                 EnvDTE.Project project = VsHelpers.DTE.SelectedItems.Item(1)?.ProjectItem?.ContainingProject;
-                project?.AddFileToProjectAsync(_configFileName);
+                project?.AddFileToProject(_configFileName);
```

Comment:
Why did we have to do this?

Reply:
@raquelsa changed this recently. Could you please pull her changes?

Reply:
Ah, thanks, good catch. That was not intended.